lib/src/actions/mod.rs

```diff
@@ -120,6 +122,9 @@ pub enum Actions {
 
     #[serde(rename = "package.repository", alias = "package.repo")]
     PackageRepository(ConditionalVariantAction<PackageRepository>),
+
+    #[serde(rename = "user.add", alias = "add.user")]
```

Comment:
Can we remove "add.user" as every other action is "module.action" and I'd like to keep it that way 👍🏻